File: CS/E4422/CustomRichEdit/CustomEditViewInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraEditors.Repository;
using DevExpress.XtraEditors.ViewInfo;

namespace CustomEditor {
    public class CustomEditViewInfo : RichTextEditViewInfo {
        public CustomEditViewInfo(RepositoryItem item)
            : base(item) {
        }

        public override string DisplayText {
            get {
                object value = Viewer.EditValue;
                if (value==null)
                    return String.Empty;
                return value.ToString();
            }
        }
    }
}

File: VB/E4422/CustomRichEdit/CustomEditViewInfo.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraEditors.Repository
Imports DevExpress.XtraEditors.ViewInfo

Namespace CustomEditor
	Public Class CustomEditViewInfo
		Inherits RichTextEditViewInfo
		Public Sub New(ByVal item As RepositoryItem)
			MyBase.New(item)
		End Sub

		Public Overrides ReadOnly Property DisplayText() As String
			Get
				Dim value As Object = Viewer.EditValue
				If value Is Nothing Then
					Return String.Empty
				End If
				Return value.ToString()
			End Get
		End Property
	End Class
End Namespace
